definitions/variable/tag_electricity_storage_types.yaml

```diff
@@ -6,10 +6,12 @@
 - Electricity Storage Type:
 
   - Energy Storage System:
-      description: Mainly to refer to the batteries
+      description: all types of energy storage including pumped hydro and batteries
+  - Energy Storage System|Pumped Hydro Storage:
```

Comment:
In my opinion, there are three main points:

- `Energy Storage System|Pumped Hydro Storage` should be removed since we already have `Hydro|Pumped Storage`.
- `Hydro|Reservoir`, `Hydro|Pumped Storage` and `Energy Storage System|*` have variables in common related to the *storage operation*.
- `Hydro|Pumped Storage` takes electricity from the grid to store water which is the main difference concerning `Energy Storage System|*`. This characteristic can be also applied to thermal energy storage for `Solar|CSP`.

So, perhaps we can create a new category for this kind of storage where the power unit takes electricity from the grid to store another energy source type that will be used to produce electricity later.

Reply:
Please take another look at the current status and #199, in particular for which variables the tags `Electricity Storage Type` vs. `Electricity Input` are used.

Also, all storage types take electricity from the grid to store it as another source of energy (gravity, kinetic, chemical, ...) and then revert it back to electriciy.

Reply:
Ok, I also agree with all storage types of energy and their variables injected/discharged.

So, I got the purpose of defining `Energy Storage System|Pumped Hydro Storage`. Since we can use the storage operation variables for it and the variables related to electricity operation by using `Hydro|Pumped Storage`. 
Both definitions for PHS are fine.

Moreover, we could perhaps define `Energy Storage System|Hydro Reservoir` in case the variables related to reservoir operation would be needed.